src/draw/lv_draw_buf.h

```diff
@@ -43,11 +43,11 @@ typedef struct {
  * Refine it to suit your needs.
  */
 
-#define _LV_DRAW_BUF_STRIDE(w, cf) \
+#define LV_DRAW_BUF_STRIDE(w, cf) \
     LV_ROUND_UP(((w) * LV_COLOR_FORMAT_GET_BPP(cf) + 7) / 8, LV_DRAW_BUF_STRIDE_ALIGN)
 
 /* Allocate a slightly larger buffer, so we can adjust the start address to meet alignment */
-#define _LV_DRAW_BUF_SIZE(w, h, cf) (_LV_DRAW_BUF_STRIDE(w, cf) * (h) + LV_DRAW_BUF_ALIGN)
+#define LV_DRAW_BUF_SIZE(w, h, cf) (LV_DRAW_BUF_STRIDE(w, cf) * (h) + LV_DRAW_BUF_ALIGN)
```

Comment:
This macro should be a function, macros are not accessible from high level language bindings

Reply:
Same as above.